actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -136,7 +136,7 @@
 {
     private Logger logger = LoggerFactory.getLogger(Stage.class);
 
-    private static final int DEFAULT_EXECUTION_POOL_SIZE = 128;
+    private static final int DEFAULT_EXECUTION_POOL_SIZE = 32;
```

Comment:
Seems way to high for the default settings of Orbit, thread size in 64bit JVM is 1024k, multiplied by 128..